### ISSUE TITLE
Keep username and Domain name synchronized in Invitations

### ISSUE THREAD

**Issue by dborovcanin** [Mainflux/mainflux-lite#1985]
### Is your feature request related to a problem? Please describe.

We should keep username and Domain name, stored in the Invitation in absmach/magistrala#1987, in sync with potential updates.

### Describe the feature you are requesting, as well as the possible use case(s) for it.

If I change the username or name of the domain, I expect all the Invitations in Invitations svc to be updated accordingly. I do not expect that from the email.

### Indicate the importance of this feature to you.

Must-have

**Comment by dborovcanin** [comment]
Please double check, this may not be needed anymore.

**Comment by nyagamunene** [comment]
> Please double check, this may not be needed anymore.

@dborovcanin I don't think this is needed because processing of the invitation is done in the UI
